ObsFacilityWikidata.tex

```diff
@@ -75,23 +76,25 @@ \section{Introduction}
 the specific previous example, a query on EPN-TAP will not show any 
 result, since the available data are registered with an 
 \texttt{instrument\_host\_name} set to \emph{HST}. It is up to the 
-data provider to provide adequate observation facility names, so 
-that users can find the resources they are looking for. However, 
+data provider to provide observation facility names that likely match
+users' ideas of their designations.  Since
 observation facility names usually have acronyms, aliases or alternate 
-names, depending on the context, or even on observation periods of 
-time (e.g., space mission can change name when they are extended). 
+names, which may further depend the context or even on 
+time (e.g., space mission can change name when they are extended), this
+is certainly no easy task.
 
-In order to address this use-case, two steps are required: (a) group
+In order to address this use case, two steps are required: (a) group
 all known aliases for each observation facilities in a lookup table, 
 which can be used to build a dedicated name resolver service; and 
-(b) propose a vocabulary, which would be used by data providers to 
-register their datasets. This document does not address another key
+(b) propose a vocabulary to be used in new data in order to obviate the
```

Comment:
A vocabulary does not remove the need for a name resolver, but it simplifies a lot the queries. Without a vocabulary: the name resolver is used to find all aliases and search for them. With a vocabulary, the name resolver is only needed to find the term to be used in the query.